### PR TITLE
[Fix][Doc] Fixed links in some documents

### PR DESCRIPTION
**Pull request by xxzuo**
### Purpose of this pull request

- Reference links have been added to the English document
- Replace some English document links in Chinese documents with translated Chinese document links

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by xxzuo** [comment]
@EricJoy2048 @liugddx  Sorry to bother, I have a question. I added a documentation link [debezium documentation](https://debezium.io/documentation/reference/2.5/connectors/mysql.html#mysql-events) in this PR, which is accessible, but it shows "dead links" in this GitHub Action.

**Comment by xxzuo** [comment]
image

**Comment by xxzuo** [comment]
> @EricJoy2048 @liugddx Sorry to bother, I have a question. I added a documentation link [debezium documentation](https://debezium.io/documentation/reference/2.5/connectors/mysql.html#mysql-events) in this PR, which is accessible, but it shows "dead links" in this GitHub Action.


Sorry, I added extra parentheses. Please ignore it.

File: docs/zh/connector-v2/source/HdfsFile.md [hunk 1]
```diff
@@ -10,15 +10,15 @@
...
-- [x] [精确一次](../../../en/concept/connector-v2-features.md)
+- [x] [批处理](../../concept/connector-v2-features.md)
+- [ ] [流处理](../../concept/connector-v2-features.md)
+- [x] [精确一次](../../concept/connector-v2-features.md)
 
 在一次 pollNext 调用中读取分片中的所有数据。将读取的分片保存在快照中。
 
-- [x] [列投影](../../../en/concept/connector-v2-features.md)
-- [x] [并行度](../../../en/concept/connector-v2-features.md)
-- [ ] [支持用户定义的分片](../../../en/concept/connector-v2-features.md)
```
Comment (Hisoka-X) [inline docs/zh/connector-v2/source/HdfsFile.md:21]: Please do not change this. It work fine in website. You can refer https://seatunnel.apache.org/docs/2.3.4/connector-v2/source/HdfsFile

Comment (xxzuo) [reply docs/zh/connector-v2/source/HdfsFile.md:21]: Thank you for your review. The website is an English document, will my modifications to the Chinese document also have an impact?

Comment (Hisoka-X) [reply docs/zh/connector-v2/source/HdfsFile.md:21]: Oh, sorry about that. I missed this is chinese file. Are you verified in local with this change? Does it work fine?

**Review by Hisoka-X — APPROVED** [review]
LGTM. Thanks @xxzuo

**Review by hailin0 — APPROVED** [review]

File: docs/zh/connector-v2/source/HdfsFile.md (hunk 1 above)
Comment (xxzuo) [reply docs/zh/connector-v2/source/HdfsFile.md:21]: Yes, I replaced the associated document link here with the translated Chinese document link.

**Pull request merged**